src/System.Management.Automation/engine/hostifaces/InternalHostUserInterface.cs

```diff
@@ -120,6 +120,47 @@ public override
             return result;
         }
 
+        /// <summary>
+        /// See base class.
+        /// </summary>
+        /// <returns>
+        /// The characters typed by the user.
+        /// </returns>
+        /// <exception cref="HostException">
+        /// If the UI property of the external host is null, possibly because the PSHostUserInterface is not
+        ///     implemented by the external host.
```

Comment:
```suggestion
        /// implemented by the external host.
```

Reply:
fixed in 198b33a4a8ee1e69fdb23b1a242458cafdb5fb7a